jobs/validate/integrator-spec

```diff
@@ -51,6 +51,8 @@ EOF
     elif [ "$JUJU_CLOUD" = "aws/us-east-1" ]; then
       # Deploy aws
       cat <<EOF >> overlay.yaml
+  easyrsa:
+    to: null
```

Comment:
what does the to: null do in this case?

Reply:
this negates any option set at this field. 

the kubernetes-core bundle sets this on `easyrsa`
```yaml
    easyrsa:
        to:
        - lxd:0
```

this means easyrsa ends up on a lxd on machine 0.  When this occurs, it can't find out information about it's public ip address that it needs to build a cluster certificate.

So -- by removing this config option it will get its own VM in the cloud rather than running in a lxd